<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrackerLab</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>225</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="openFilesButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Open Files...</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="frameSpinBox">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>97</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="fileLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>57</y>
         <width>47</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Files:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="fileComboBox">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>54</y>
         <width>281</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="frameLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>101</y>
         <width>47</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Frame:</string>
       </property>
      </widget>
      <widget class="QSlider" name="frameSlider">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
      <widget class="QLabel" name="infoLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>195</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>x = 0    y = 0    [0]</string>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>30</y>
         <width>561</width>
         <height>181</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab1">
        <attribute name="title">
         <string>Measure Labels</string>
        </attribute>
        <widget class="QDoubleSpinBox" name="tab1ThresholdSpinBox">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>20</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.100000000000000</double>
         </property>
        </widget>
        <widget class="QLabel" name="tab1ThresholdLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Threshold:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="tab1MinAreaLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Min. Area:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="tab1MaxAreaLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Max. Area:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QSpinBox" name="tab1MinAreaSpinBox">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>50</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="tab1MaxAreaSpinBox">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>80</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>250</number>
         </property>
        </widget>
        <widget class="QCheckBox" name="tab1InvertCheckBox">
         <property name="geometry">
          <rect>
           <x>153</x>
           <y>22</y>
           <width>91</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Invert</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab2">
        <attribute name="title">
         <string>Difference of Gaussians</string>
        </attribute>
        <widget class="QDoubleSpinBox" name="tab2ThresholdSpinBox">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>20</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.100000000000000</double>
         </property>
        </widget>
        <widget class="QSpinBox" name="tab2MaxSigmaSpinBox">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>50</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
        <widget class="QLabel" name="tab2ThresholdLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Threshold:</string>
         </property>
        </widget>
        <widget class="QLabel" name="tab2MaxSigmaLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Max. Sigma:</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QPushButton" name="batchButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>10</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Batch</string>
       </property>
      </widget>
      <widget class="QPushButton" name="abortButton">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
      <widget class="QLabel" name="numberOfSpots">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>192</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="numberOfSpotsLabel">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>191</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string> # of Spots:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QFrame" name="preprocessingFrame">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>30</y>
         <width>161</width>
         <height>181</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <widget class="QCheckBox" name="medianCheckBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>70</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Median:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="medianSpinBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
       </widget>
       <widget class="QCheckBox" name="maskCheckBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>70</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Mask:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="maskSpinBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>90</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="preprocessingFrameLabel">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>10</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pre-Processing</string>
       </property>
      </widget>
      <widget class="QLabel" name="propertiesLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="trackingCheckBox">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>10</y>
         <width>231</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string> Tracking</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
